parm/config/config.resources

```diff
@@ -202,6 +203,71 @@ elif [[ "${step}" = "atmanlfinal" ]]; then
     export npe_node_atmanlfinal
     export is_exclusive=True
 
+elif [[ "${step}" = "landanlinit" || "${step}" = "landanlfinal" ]]; then
+
+   # below lines are for creating JEDI YAML
+   case ${CASE} in
+     C768)
+        layout_x=6
```

Comment:
are these layouts chosen arbitrarily? @jiaruidong2017 do you have a feel for how resolution affects runtime of the JEDI OI code?

Reply:
@CoryMartin-NOAA These layouts were chosen just following the aero's layouts. I didn't have any tests yet for how resolution affects runtime of the JEDI IO. Thanks.